controller.go

```diff
@@ -163,7 +165,7 @@ type initializer struct {
 }
 
 // New creates a new instance of network controller.
-func New(cfgOptions ...config.Option) (NetworkController, error) {
+func New(getter getter.PluginGetter, cfgOptions ...config.Option) (NetworkController, error) {
```

Comment:
`getter.PluginGetter` can be one of the config.Option. That will save the `New` API from changing.